.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseMergedDateTimeExtractor.cs

```diff
@@ -327,7 +327,23 @@ private List<ExtractResult> AddMod(List<ExtractResult> ers, string text)
         {
             foreach (var er in ers)
             {
-                var success = TryMergeModifierToken(er, config.BeforeRegex, text);
+                var success = TryMergeModifierToken(er, config.BeforeAroundRegex, text);
```

Comment:
Why do we need to match both BeforeAroundRegex and BeforeRegex later? This is basically the same as before and there's no benefit in keeping both (it would be better to keep combined).
Can't we just match on around at the entity level, then add mod would only match on before/after/etc, as the method name implies. This would also allow just the around + datetime form to always work, even if no relative prefix.

Reply:
Ah, I see what you mean now, yes, it makes more sense. I thought this was what you implied when saying to keep the regexes separated. I will update the code.

Reply:
code updated, 'around' modifier is still matched in Merged instead that at the entity level, but the other modifiers (before/after...) are now allowed to match also when 'around' is found. In this way, both around mod ('approx') and mixed mods (e.g. 'before-approx') can be caught.